Tests/L0/SonarQubePostTest/PRCA/OrchestratorTests.ps1

```diff
@@ -43,8 +43,8 @@ $comment2 = $comments | Where-Object {$_.line -eq 15}
 Assert-AreEqual "some/path" $comment1.RelativePath "Expecting the RelativePath to be set"
 Assert-AreEqual "some/other/path" $comment2.RelativePath "Expecting the RelativePath to be set"
 
-Assert-AreEqual "CA issue 1" $comment1.Content "Expecting the Content to be set"
-Assert-AreEqual "CA issue 2" $comment2.Content "Expecting the Content to be set"
+Assert-AreEqual "CA issue 1 (RemoveUnusedLocals)" $comment1.Content "Expecting the Content to be set"
```

Comment:
This is how the rule ID is going to be presented. Alternatively it could be a prefix, but I tend to like it more like this.